docs/install/knative-with-operators.md

```diff
@@ -38,18 +86,23 @@ Information about Knative Serving Operator releases can be found on the [Release
 
 __From releases__:
 
-Replace \<version\> with the latest version or the version you would like to install, and run the following command to
-install Knative Serving Operator:
+Run the following command to install Knative Serving Operator:
 
 ```
-kubectl apply -f https://github.com/knative/serving-operator/releases/download/<version>/serving-operator.yaml
+kubectl apply -f https://github.com/knative/serving-operator/releases/download/v0.14.0/serving-operator.yaml
 ```
 
 __From source code__:
```

Comment:
Here's a thought - could all of this be rewritten to use tabs please instead like the networking layer part here? https://knative.dev/docs/install/any-kubernetes-cluster/
I don't see why we would have these differently formatted headings here instead.